cppcache/include/geode/internal/DSCode.hpp

```diff
@@ -31,12 +31,10 @@ enum class DSCode : int32_t {
   FixedIDShort = 2,
   CacheableLinkedList = 10,
   Properties = 11,
-  CacheableToken = 14,  // because there's no equivalence in java
   PdxType = 17,  // internal hack to read pdxtype in c# layer, look usuage in
```

Comment:
I think `PdxType` can go away. It isn't really a `DSCode`.

Reply:
It's still used by `DistributedSystem::AppDomainInstanceInitialization` and `TcrMessage`. Will address in separate ticket: https://issues.apache.org/jira/browse/GEODE-5743